cypress/support/fragments/settings/data-export/fieldMappingProfiles.js

```diff
@@ -0,0 +1,124 @@
+            MultiColumnListHeader('Transformation').exists(),
+        ])
+    },
+    verifySearchAndFilterPane() {
+        this.verifyAllSearchAndFilterCheckboxesChecked();
+        this.verifyTransformationsPaneColumns();
+    },
+    searchText(text) {
+        cy.do(transformationsSearchTextfield.fillIn(text));
+        cy.get('input[name="searchValue"]').type('{enter}');
+    },
+    clickResetAll() {
+        cy.do(resetAllButton.click());
+    },
+    uncheckInstanceRecordTypeChechbox() {
+        cy.expect(instanceRecordTypeChechbox.has({ checked: true }));
+        cy.do(instanceRecordTypeChechbox.click());
+    },
+    uncheckHoldingsRecordTypeChechbox() {
+        cy.expect(holdingsRecordTypeChechbox.has({ checked: true }));
+        cy.do(holdingsRecordTypeChechbox.click());
+    },
+    uncheckItemRecordTypeChechbox() {
+        cy.expect(itemRecordTypeChechbox.has({ checked: true }));
+        cy.do(itemRecordTypeChechbox.click());
+    },
+    uncheckSelectedStatusChechbox() {
+        cy.expect(selectedStatusChechbox.has({ checked: true }));
+        cy.do(selectedStatusChechbox.click());
+    },
+    uncheckUnselectedStatusChechbox() {
+        cy.expect(unSelectedStatusChechbox.has({ checked: true }));
+        cy.do(unSelectedStatusChechbox.click());
+    },
+    checkInstanceRecordTypeChechbox() {
+        cy.expect(instanceRecordTypeChechbox.has({ checked: false }));
+        cy.do(instanceRecordTypeChechbox.click());
+    },
+    checkHoldingsRecordTypeChechbox() {
+        cy.expect(holdingsRecordTypeChechbox.has({ checked: false }));
+        cy.do(holdingsRecordTypeChechbox.click());
+    },
+    checkItemRecordTypeChechbox() {
+        cy.expect(itemRecordTypeChechbox.has({ checked: false }));
+        cy.do(itemRecordTypeChechbox.click());
+    },
+    checkSelectedStatusChechbox() {
+        cy.expect(selectedStatusChechbox.has({ checked: false }));
+        cy.do(selectedStatusChechbox.click());
+    },
+    checkUnselectedStatusChechbox() {
+        cy.expect(unSelectedStatusChechbox.has({ checked: false }));
+        cy.do(unSelectedStatusChechbox.click());
+    },
+    verifyTotalSelected(expectedTotalSelected) {
+        cy.expect(Modal('Select transformations').has({ content: including(`Total selected: ${expectedTotalSelected}`) }));
+    },
+    fillInTransformationsTextfields(textfield1, textfield2, textfield3, textfield4, rowIndex = 0) {
+        // Couldn't get the element with interactors
+        cy.xpath(`//div[contains(@class, "mclRow--")][${rowIndex + 1}]//input[contains(@name, "marcField")]`).type(textfield1);
+        cy.xpath(`//div[contains(@class, "mclRow--")][${rowIndex + 1}]//input[contains(@name, "indicator1")]`).type(textfield2);
+        cy.xpath(`//div[contains(@class, "mclRow--")][${rowIndex + 1}]//input[contains(@name, "indicator2")]`).type(textfield3);
+        cy.xpath(`//div[contains(@class, "mclRow--")][${rowIndex + 1}]//input[contains(@name, "subfield")]`).type(textfield4);
```

Comment:
You can find some functions what can use instead cy.xpath in cypress\support\fragments\data-export\exportMappingProfile\modalSelectTransformations.js

Reply:
it cannot be implemented as the page has multiple rows and they don't work with interactors for now